docs/_posts/0000-05-01-oo-inheritance.md

```diff
@@ -131,24 +110,30 @@ Work in language groups to collaboratively come up with a diagram
 
 ## Exercise Review and Implementation
 
-```text
+<mermaid>
+classDiagram
+    Card <|-- AnimalCard : implements
+    Card <|-- PlayingCard : implements
+    class PlayingCard {
+        snap(Card)
+    }
+    class AnimalCard {
+        snap(Card)
+    }
+    Deck <|-- AnimalDeck : implements
+    Deck <|-- PlayingCardDeck : implements
```

Comment:
As before, this should be inheritance:
```suggestion
    Deck <|-- AnimalDeck
    Deck <|-- PlayingCardDeck
```

Reply:
Thanks! I’ll change these to `extends` rather than `implements`.